src/metrics/rules/validator/validator.go

```diff
@@ -196,6 +199,54 @@ func (v *validator) validateRollupRules(rrv []view.RollupRule) error {
 	return validateNoDuplicateRollupIDIn(pipelines)
 }
 
+func (v *validator) validateUtilziationRules(rrv []view.UtilizationRule) error {
```

Comment:
Sigh this is so painful. Another perfect case where generics in Go would help here 😕